yamoveset/panels/main.py

```diff
@@ -280,10 +280,14 @@ def get_changed_values(
 
     def on_paste(self, _):
         if not self.parent.copied:
+            with wx.MessageDialog(self, f'No entries are copied from the right panel to Paste') as dlg:

```

Comment:
little known fact.  It doesn't have to be from the same copy of YaMoveset.  You can actually have a copy still sitting in the clipboard from another YaMoveset instance.  I'd rather just disable the button, but I'll let this slide for now.